helpers/mail.ts

```diff
@@ -2,7 +2,7 @@ import _ from 'lodash'
 import mailgun from 'mailgun-js'
 
 const CLIENT_URL = _.get(process, 'env.CLIENT_URL', 'https://c0d3.devwong.com')
-const MAILGUN_API_KEY = _.get(process, 'env.MAILGUN_API_KEY', '')
+const MAILGUN_API_KEY = _.get(process, 'env.MAILGUN_API_KEY', 'test-key')
```

Comment:
There was already default `''` value, so I changed it a bit to simplify testing. Otherwise hoisting would complicate imports by a lot (to import schema you need to have valid API_KEY, but you can't define it before imports because every import is moved to the top).